tests/tkupchyn/homework_04/test_string_methods.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+def test_format_string():
+    pattern = "{second} {first}".format(first="world", second="hello", other="some")
```

Comment:
formatting already happened here so next 2 lines do not do much, may be it was intended?